sensor_msgs/LaserScan front
sensor_msgs/LaserScan left
sensor_msgs/LaserScan right